crates/gpui/src/platform/windows/platform.rs

```diff
@@ -314,12 +317,25 @@ impl Platform for WindowsPlatform {
         }
     }
 
-    // todo(windows)
-    fn activate(&self, _ignoring_other_apps: bool) {}
+    fn activate(&self, _ignoring_other_apps: bool) {
+        let mut state = self.hidden_windows.write();
+        state.iter().for_each(|handle| unsafe {
+            ShowWindow(*handle, SW_SHOW).ok().log_err();
```

Comment:
Here maybe need to improve to support `_ignoring_other_apps` option, I will try other option for `ShowWindow` method.

https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-showwindow

Reply:
I think that is a [macOS specific](https://developer.apple.com/documentation/appkit/nsapplication/1428468-activateignoringotherapps) behavior, so it should be ok to drop this.